src/appsco-app.html

```diff
@@ -1725,6 +1735,11 @@
                     value: ''
                 },
 
+                _importCustomerResourcesApi: {
```

Comment:
This should be computed of _companyCustomersApi. Similar to _companyImportContactsApi.